ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -39,21 +39,29 @@ public ClinicalDetailsModel(INotificationService service, NtbsContext context, I
             this.auditService = auditService;
         }
 
-        public override async Task<IActionResult> OnGetAsync(int? id)
+        public override async Task<IActionResult> OnGetAsync(int? id, bool isBeingSubmitted)
```

Comment:
When redirected from error page there is no other way to know that Notification is being submitted without passing it in request. Alternative would be to have another transition status (NotificationStatus) but this would be overkill and we probably don't want to save it in DB